geoparquet/index.qmd

```diff
@@ -103,4 +103,23 @@ Parquet supports a very extensive type system, including nested types such as li
 
 ## References
 
-- [Demystifying the Parquet File Format](https://towardsdatascience.com/demystifying-the-parquet-file-format-13adb0206705)
\ No newline at end of file
+- [Demystifying the Parquet File Format](https://towardsdatascience.com/demystifying-the-parquet-file-format-13adb0206705)
+
+<!-- Footnotes: -->
+
+[^parquet-via-gdal]: As [pointed out by GDAL developer Even Rouault](https://github.com/rocker-org/rocker-versioned2/pull/692#issuecomment-1703201670), reading GeoParquet through GDAL is just as fast as reading through the `geopandas.read_parquet` function **_if_** you're using GDAL's Arrow API. As of September 2023, this is not the default, so you need to opt into the `pyogrio` engine and opt into the Arrow API:
+
+    ```py
+    import geopandas as gpd
+    gpd.read_file("file.parquet", engine="pyogrio", use_arrow=True)
+    ```
+
+    It's also necessary to note that the Python wheels distributed by pyogrio do
+    not include the Arrow and Parquet drivers by default. In order to use the
+    `pyogrio` driver for a GeoParquet file, you need to compile from source when
+    installing. You'll need to have GDAL on your computer already, and then you
+    can install `pyogrio` via:
+
+    ```bash
+    pip install pyogrio --no-binary pyogrio
```

Comment:
No, this requires the GDAL installation on your machine to be built against Arrow

Reply:
What's the GDAL min version, and how would a user currently do that (compile from source?)

Reply:
I was trying to avoid getting into the weeds of how to install GDAL (since this is already in an advanced footnote). Right now you can use homebrew or conda; on linux you have to compile from source I think. Do you want me to expand on this on the footnote?

Reply:
I would specify the min GDAL version, and say that it has to have the arrow driver (as seen with ogrinfo --formats). No need to explain installing if any common method works.

Reply:
> No need to explain installing if any common method works.

Well, this gets back to your comment about Ubuntu packaging 🙂 . The LTS GDAL version is [too old at 3.5](https://packages.ubuntu.com/kinetic/libgdal-dev). You have to use [ubuntugis-unstable](https://launchpad.net/~ubuntugis/+archive/ubuntu/ubuntugis-unstable) to get 3.6.

That said, I do think I'd like to avoid getting into the weeds of that in this footnote at least

Reply:
You already put September 2023, just add (gdal >= 3.6)

Reply:
just like COG need GDAL > 3.1 (which still comes on some older distros ....)

Reply:
I added 

> You'll need to have a GDAL installation version 3.6 or later

on Friday so I think this should be good to merge. Let me know if you have more suggestions and I can add a follow-up PR